graphicsmagick/src/wand/magick.rs

```diff
@@ -2338,11 +2327,13 @@ impl<'a> MagickWand<'a> {
     ///
     /// MagickQueryFonts() returns any font that match the specified pattern.
     ///
-    pub fn query_fonts(pattern: &str) -> Option<Vec<Result<String, Utf8Error>>> {
+    pub fn query_fonts(pattern: &str) -> Option<Vec<Option<MagickCString>>> {
```

Comment:
Comment:

I will post another PR addressing functions returning a `Vec` to also make them zero-cost.